cajita/src/Cajita_FastFourierTransform.hpp

```diff
@@ -23,291 +23,509 @@
 #include <memory>
 #include <type_traits>
 
+#ifdef KOKKOS_ENABLE_HIP // FIXME_HIP
+#error FFT not supported with Kokkos::Experimental::HIP backend
```

Comment:
This error should be a `static_assert` in the HIP specialization instead of a macro-level error here. That way it would only fail when people actually tried to use it. The way this is written here it will possibly preclude users from having HIP kernels and an FFT on the host.

Reply:
Done